ord_schema/validations.py

```diff
@@ -256,6 +358,10 @@ def validate_reaction_outcome(message):
                 if key not in analysis_keys:
                     raise ValueError(f'Undefined analysis key {key} '
                                      'in ReactionProduct')
+    if not message.products:
+        warnings.warn('No products specified for reaction outcome; this should'
+                      ' only be used when screening for reaction conversion',
```

Comment:
"this should only be used" is a bit confusing, since you really mean "this should only be empty", right?

Reply:
Ah yes I see the confusion